app/models/package_manager/pypi.rb

```diff
@@ -123,14 +123,14 @@ def self.known_versions(name)
         &.index_by { |v| v[:number] } || {}
     end
 
-    def self.dependencies(name, version, _mapped_project)
+    def self.dependencies(name, version, _mapped_project = nil)
```

Comment:
Not sure we need this change. Is something calling this method with passing in the third argument?

Reply:
Just the test.